radix-engine-stores/src/hash_tree_support.rs

```diff
@@ -12,15 +12,17 @@ pub struct HashTreeUpdatingDatabase<D> {
     tree_store: TypedInMemoryTreeStore,
     current_version: Version,
     current_hash: Hash,
+    jmt_enabled: bool,
```

Comment:
Not sure if this is the right place for the flag. Feels like if we want to disable jmt we should just not use the `HashTreeUpdatingDatabase`.

Reply:
I had the same thought initially, then I realised that I would have to track progress manually (which is already done by `HashTreeUpdatingDatabase`). I then viewed it as `VersionedSubstateDatabase`.

Since executing transaction without JMT is mostly for testing/anayzing (instead of state hash computation), I will remove this flag completely. I will make dedicated command for other purposes.

Reply:
Done